Cargo.toml

```
@@ -1,10 +1,11 @@
 [patch.crates-io]
 serial = { git = "https://github.com/bytebeamio/serial-rs", branch = "android_fix" }
 
+
 [workspace]
 members = [
     "uplink",
-    "disk",
+    "disk"
```

```suggestion
    "disk",
```

This is the oxford comma, it's a good practice to include it in YAML